scripts/deploy.sh

```diff
@@ -22,7 +22,23 @@ set -o xtrace
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 
 cd ${SCRIPT_ROOT}
+# Dedicated namespace has to be present beforehand.
 kubectl apply -f manifests/0-namespace.yaml
+
+# Generate Secret with dynamically initialized data, so as to keep Go codebase
+# simple as possible i.e., without client-go ClientSet.
+cat <<EOF | kubectl create -f -
```

Comment:
Please create a db-secret.yml not generating a key in deploy.sh.
The deploy.sh should be a simple support script.

Reply:
Thanks for the comment. If we still keep the password randomised as my original patch does, I may choose either of:

- (1). make it as a go template, add a lightweight cli tool to do `template.Execute()`
- (2). choose a popular template engine, such as jinja2

Does the option (1) sound ok to you? If I misunderstood your point, please correct me.

Reply:
Do we really need to randomize?
Why not to make secret manifest and set a default password. Then users will edit it when they need.